frontend/web/components/EnvironmentDocumentCodeHelp.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useState } from 'react'
+import Constants from 'common/constants'
+import CodeHelp from './CodeHelp'
+import Icon from './Icon'
+import Tabs from './base/forms/Tabs'
+import TabItem from './base/forms/TabItem'
+import InfoMessage from './InfoMessage'
+import { useGetServersideEnvironmentKeysQuery } from 'common/services/useServersideEnvironmentKey'
+type EnvironmentDocumentCodeHelpType = {
+  environmentId: string
+  title: string
+}
+
+const EnvironmentDocumentCodeHelp: FC<EnvironmentDocumentCodeHelpType> = ({
+  environmentId,
+  title,
+}) => {
+  const [visible, setVisible] = useState(false)
+  const { data } = useGetServersideEnvironmentKeysQuery({ environmentId })
+  return (
+    <div>
+      <div
+        style={{ cursor: 'pointer' }}
+        onClick={() => {
+          setVisible(!visible)
+        }}
+      >
+        <Row>
+          <Flex style={isMobile ? { overflowX: 'scroll' } : {}}>
+            <div>
+              <pre className='hljs-header'>
+                <span />
+                {'<>'} Code example:{' '}
+                <span className='hljs-description'>{title}</span>
+                <span className='hljs-icon'>
+                  <Icon
+                    name={visible ? 'chevron-down' : 'chevron-right'}
+                    width={16}
+                  />
+                </span>
+              </pre>
+            </div>
+          </Flex>
+        </Row>
+      </div>
+      {!!visible && (
+        <div className='mt-2'>
+          <InfoMessage>
+            <div>
+              Providing flag defaults is recommended for{' '}
+              <a
+                target='_blank'
+                href='https://docs.flagsmith.com/guides-and-examples/defensive-coding'
+                rel='noreferrer'
+              >
+                defensive coding
+              </a>{' '}
+              and allowing offline capabilities.
+              <br />
+              By default SDKs run in remote evaluation mode, server-side SDKs
+              can also run in local evaluation mode.{' '}
```

Comment:
The reasoning here was to support _offline_ mode which is kind of a subset of local evaluation mode. I think here we want to specify that, right?

Reply:
Hmm we're mentioning this twice in the UI


<img width="927" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/8608314/b4798e48-66ec-42a0-b4f9-e3dbae4d3924">